.gitmodules

```diff
@@ -5,6 +5,7 @@
 [submodule "libgit2"]
 	path = thirdparty/git2/libgit2
 	url = https://github.com/libgit2/libgit2
+	ignore = untracked
```

Comment:
Out of curiosity, why is this change needed?

Reply:
This was a year ago, I think adding that stopped git from telling me there were changes in the libgit2.  When I rebase against the latest master, I'll see if it was really necessary or just an artifact of me trying to figure out the problem.